packages/vscode-ui5-language-assistant/package.json

```diff
@@ -79,15 +79,15 @@
     "vscode": "^1.47.0"
   },
   "scripts": {
-    "ci": "npm-run-all clean compile lint test:unit coverage bundle package",
+    "ci": "npm-run-all clean compile lint coverage:* bundle package",
```

Comment:
Yes it would be but currently the integration tests are disabled and run "empty".
So i made sure the `unit` ones are the "last" to run so we at least get the report for that.
Fixing the coverage (that never worked for the VSCode E2E tests) is outside the scope of this PR.